docs/CONFIGURATION.md

```diff
@@ -422,6 +422,39 @@ after_request_complete do |server, worker, env|
 end
 ```
 
+### `before_service_worker_ready` (experimental)
+
+Experimental and may change at any point.
+
+If defined, Pitchfork will spawn one extra worker, called a service worker
+which doesn't accept incomming requests, but allows to perform service tasks
```

Comment:
Had to scratch my Ruby itch, sorry for the drive-by review 😅

```suggestion
which doesn't accept incoming requests, but allows to perform service tasks
```

Reply:
❤️